lib/runner.js

```diff
@@ -314,6 +314,9 @@ Runner.prototype.hook = function (name, fn) {
             suite.tests.forEach(function (test) {
               test.pending = true;
             });
+            suite.suites.forEach(function (suite) {
```

Comment:
This is for `beforeEach`/`afterEach` -- **not** what we wanted to do, right?

Reply:
Currently, my changes are based off these criteria (also listed above).

If a `before` is skipped:
- run any additional before / after hooks at the same level
- skip any beforeEach / afterEach hooks at the same level
- skip all tests at the same level or in nested suites
- skip all hooks in nested suites

I've been operating under the assumption that `before` hooks are executed prior to `beforeEach` hooks, which would mean that there's no context in which we'd still need to execute `beforeEach` in a skipped suite. Is that the case?

Reply:
Sorry, my bad--was looking at the wrong code.  Yes, we are in agreement.